apps/ledger-live-mobile/src/newArch/features/Market/components/DeltaVariation/index.tsx

```diff
@@ -14,13 +14,22 @@ function DeltaVariation({ value, percent, ...props }: Props) {
   }
 
   const absDelta = Math.abs(delta);
+  const roundedDelta = parseFloat(delta.toFixed(2));
+
+  if (roundedDelta === 0) {
+    return (
+      <Text variant={"large"} color="neutral.c60" fontWeight={"semiBold"}>
+        -
+      </Text>
+    );
+  }
 
   const [color, ArrowIcon, sign] =
-    delta !== 0
-      ? delta > 0
-        ? ["success.c50", IconsLegacy.ArrowEvolutionUpMedium, "+"]
-        : ["error.c50", IconsLegacy.ArrowEvolutionDownMedium, "-"]
-      : ["neutral.c100", null, ""];
+    roundedDelta > 0.0
+      ? ["success.c50", IconsLegacy.ArrowEvolutionUpMedium, "+"]
+      : roundedDelta < 0.0
+        ? ["error.c50", IconsLegacy.ArrowEvolutionDownMedium, "-"]
+        : ["neutral.c70", null, ""];
```

Comment:
```suggestion
        : ["neutral.c70", null, "-"];
```

We want to be iso as wallet page no? :)